save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/prograssbar/ReactCircle.kt

```diff
@@ -0,0 +1,68 @@
+@file:Suppress("HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE")
+@file:JsModule("react-circle")
+@file:JsNonModule
+
+package com.saveourtool.save.frontend.externals.prograssbar
+
+import react.Component
+import react.PropsWithChildren
+import react.ReactElement
+import react.State
+
+/**
+ * External declaration of [ReactCircleProps] react component
+ */
+@JsName("default")
+external class ReactCircle : Component<ReactCircleProps, State> {
```

Comment:
@sanyavertolet, as FE expert -- `Component` is okay or better to use `FC`, `VFC` and etc?

Reply:
discussed offline -- looks like it cannot be rewritten to functional component is due to internal implementation in js